src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx

```diff
@@ -472,7 +471,7 @@ Details:
     * Pro
     * Enterprise
 
-      A New Relic organization can only be on one edition at a time. Your Order will indicate your edition.
+    A New Relic organization can only be on one edition at a time. Your Order will indicate your edition.
```

Comment:
@zuluecho9 Feels odd to capitalize "Order"

Reply:
those are the license docs, and they do all sorts of legal-speak things like that